src/Renderer/ThreeExtended/Feature2Texture.js

```diff
@@ -28,7 +39,6 @@ function drawPolygon(ctx, vertices, indices, origin, scale, properties, style =
             _lineTo(ctx, vertices[indice.offset + j], scale, origin);
         }
     }
-    ctx.closePath();
```

Comment:
Are you sure about this removal?

Maybe it's not needed because the polygons we draw are guaranteed to be closed? (but I'd keep it to be on the safe side anyway)...

Reply:
Yes this is a fix needed because not closing the path would create visual artifacts for `multipolygon`. I missed that when working on the geojson refactoring last time, sorry about that.